lib/Cake/Test/Case/Network/Email/CakeEmailTest.php

```diff
@@ -2449,4 +2449,23 @@ public function assertLineLengths($message) {
 		}
 	}
 
+/**
+ * Test if the EmailConfig::$default configuration is read when present
+ *
+ * @return void
+ */
+	public function testDefaultConfig() {
+		$defaultConfig = new File(APP . 'Config' . DS . 'email.php.default');
+		$emailConfig = new File(APP . 'Config' . DS . 'email.php');
+		$hasConfig = $emailConfig->exists();
+		$this->skipIf(!$defaultConfig->copy(APP . 'Config' . DS . 'email.php', false));
```

Comment:
This might end up overwriting someone's email config in their app if they run the core tests.